### ISSUE TITLE
Proposal: <podcast:live>

### ISSUE THREAD

**Issue by PofMagicfingers** [Podcastindex-org/podcast-namespace#181]
*This is a transfered proposal from our former project podCloud/podcast-ext. See #173 for details.*

Original proposal by @Bigaston.

**<podcast:live type="[mime type]" href="[url]">**

This could be kind of the same as og:url or twitter card. You could provide multiple urls, some with `type="text/html"` for embed, but also `type="audio/mp3"` if you have any kind of livestream channel (icecast, etc)

**Comment by Bigaston** [comment]
Maybe we can add title/description for the value of the tag? Like

`<podcast:live type="[mime type]" href="[url]">Some title</podcast:live>`

**Comment by swschilke** [comment]
As it mentions og:url isn't that something similar to Dublin Core?

**Comment by PofMagicfingers** [comment]
og stands for opengraph. Here is an example of it :
https://github.com/niallkennedy/open-graph-protocol-examples/blob/master/video-movie.html

Of course in our case, this should work with only one tag, allowing to
reference a livestream related to the podcast.

As it mentions og:url isn't that something similar to Dublin Core?

**Comment by agates** [comment]
Think it would be feasible to [somehow] utilize #174 with this to avoid duplicating source file work?

**Comment by PofMagicfingers** [comment]
IMHO this one would be inserted on `channel` level, and `podcast:media` seems to be more relevant mostly for items (but maybe I didn't catch everything reading the issue).
Maybe we could use `podcast:media` on channel to add livestream info. 

I liked the fact podcast:live was specifically for live viewing and could point to a website, a Twitch channel, a icecast livestream or pretty much anything.

**Comment by agates** [comment]
Yes I'm primarily thinking of `podcast:source`.  Example of what it might look like:

```xml
<podcast:live type="[mime type]" title="Some title">
    <podcast:source url="https://example.com/embed.html" />
    <podcast:source url="ipfs://linkForHtmlFile" />
</podcast:live>
```

This way the ability to retain different transfer protocols is still in place.  But I may just be overthinking it :)

**Comment by daveajones** [comment]
I can see the benefit of both approaches. I’m not sure which one makes the most sense. Expanding #174 to the channel level feels good. But if the purpose of this tag is so specific it would also make sense to have it be unique.

I think we need more voices on this probably.   The rawvoice namespace has a live element I think. @cio-blubrry any thoughts?

**Comment by benjaminbellamy** [comment]
I like @PofMagicfingers 's approach: it's very dreadfully simple, yet extremely efficient.  
I would add an optional link to an ics file:
`<podcast:live type="[mime type]" href="[url]" schedule="[url]">Some title</podcast:live>`

**Comment by saerdnaer** [comment]
Related: https://github.com/Podcastindex-org/podcast-namespace/issues/212

**Comment by Muppet1856** [comment]
I'm trying to figure out how to manage live feeds of supplemental information.  Things like GRCs picture of the day, or Andrew Horowitz's graph's during DHUnplugged.  I see podping helping with this, and maybe there is a mechanism to pass an image URL through PodPing.  Thoughts?  Something like a display url with an on switch and an off switch in the custom json?

`{
  "livedisplay": {
    "display": "on" | "off",
    "url": "https://something.com/item.jpg"
  }
}`

The real challenge here is how an "average" podcaster pushes the information out.  There needs to be a simple tool for it.

**Comment by brianoflondon** [comment]
Very interesting and certainly something that could be done but with a lot of parts in the chain needed.

**Comment by yoshimo** [comment]
This would be nice for unifying apps that radio stations ship.
How would you display the show schedule, the current show and its host as well as the played music tracks in this context?

**Comment by Marzal** [comment]
> This would be nice for unifying apps that radio stations ship. How would you display the show schedule, the current show and its host as well as the played music tracks in this context?

It's not already implemented on https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/tags/live-item.md ?
